cg/cli/delete/illumina_sequencing_run.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from cg.constants.cli_options import DRY_RUN
+from cg.services.illumina.post_processing.housekeeper_storage import (
+    delete_sequencing_data_from_housekeeper,
+)
+
+
+@click.command("illumina-run")
+@DRY_RUN
+@click.option(
```

Comment:
Would it make more sense to have this as a `click.argument`? Difference would be `cg delete illumina-run <flow_cell_id>` vs `cg delete illumina-run --flow-cell-id <flow_cell_id>`

Reply:
I think production would say yes to this 👍 Thoughts @Clinical-Genomics/prodbioinfo ?+

Reply:
Agree!

Reply:
^

Reply:
Fixed